### DISCUSSION TITLE
Application not accessible in Nigeria

### DISCUSSION THREAD

**Discussion by UlrichMD** [vercel/next.js#68931] [Help]
### Summary

I have deployed a Next JS app on Vercel with a domain name from Hostinger. It works just fine in Europe and the US, but doesn't work in Nigeria. I get a TimeOut Error when I try to visit the site. Any ideas?

**Comment by raymondmbam** [comment]
Any update on this yet?

**Comment by Ovodo** [comment]
Having the same issue
